src/components/cluster/detail/Versions.js

```diff
@@ -5,6 +5,23 @@ import PropTypes from 'prop-types';
 import React, { useRef } from 'react';
 import RefreshableLabel from 'UI/refreshable_label';
 import ReleaseDetailsModal from 'modals/release_details_modal';
+import styled from '@emotion/styled';
+
+const ReleaseDetail = styled.span`
+  text-decoration: underline;
+  cursor: pointer;
+  color: ${({ theme }) => theme.colors.white2};
+
+  &:hover {
+    color: ${({ theme }) => theme.colors.white1};
```

Comment:
LGTM 

I think it would be interesting to have it extracted in the theme as something more semantic as  eg `textColor` and `hoveredTextColor` or something like that, so we can be easily consistent when applying styles.

Reply:
I'm in favour of textColor and hoveredTextColor, that's a great idea.

by the way, I think themeing and how we go forward with that could use a discussion between us 3 to make sure we're on the same page. Right now i haven't really thought much more about it yet, and as you can see we have lots of these colors now, and giving them some more meaning would be cool.

For this, it would be nice though if textColor references `white2` instead of defining the same hex value again.

Reply:
I think we had an issue opened than we can use as a basis for further discussion, and Marian is willing to work a little bit on theming as well: https://github.com/giantswarm/giantswarm/issues/7346